Makefile

```diff
@@ -96,6 +104,17 @@ push-public: clean build-debian
 		docker push ${REPOSITORY}/cp-$${component}:${CP_VERSION} || exit 1; \
   done
 
+push-nexus:
```

Comment:
What's the push-private used for? Feels like we should be using that for nexus by defininig DOCKER_REMOTE_REPOSITORY in env to avoid having to put nexus repo info in here as this is available for public use?